src/excmds.ts

```diff
@@ -3281,30 +3281,38 @@ export async function yankimage(url: string): Promise<void> {
 
 /** Change active tab.
 
-    @param index
-        Starts at 1. 0 refers to last tab of the current window, -1 to penultimate tab, etc.
+    @param id
+        A string following the following format: "[0-9]+.[0-9]+", the first number being the index of the window that should be selected and the second one being the index of the tab within that window.
```

Comment:
Somewhere in this I'd say that taball has completions for this so you don't need to guess window IDs etc